src/PrestaShopBundle/Resources/views/Admin/Translations/overview.html.twig

```diff
@@ -35,7 +35,9 @@
         var data = {
             baseUrl: '{{ app.request.getBasePath() }}',
             locale: '{{ app.request.query.get('locale') }}',
-            selected: '{{ 'modules' == app.request.query.get('type') ? theme.name : app.request.query.get('selected') }}',
+            type: '{{ app.request.query.get('type') }}',
+            currentTheme: {{ theme.name|json_encode|raw }},
```

Comment:
watchout with raw.
```suggestion
            currentTheme: '{{ theme.name|escape('js') }}',
```

Reply:
That won't work with null

Reply:
If it's not working with null.
```suggestion
            currentTheme: {% if theme.name === null %}null{% else %}'{{ theme.name|escape('js') }}'{% endif %},
```

Reply:
Looks overcomplicated when json_encode is build exactly for that purpose: it outputs the right Javascript syntax for whatever value you throw at it.